java/api/src/main/java/io/ray/api/options/ActorCreationOptions.java

```diff
@@ -11,6 +11,9 @@
 
 /** The options for creating actor. */
 public class ActorCreationOptions extends BaseTaskOptions {
+  public static final int NO_RESTART = 0;
+  public static final int INFINITE_RESTART = (int) Math.pow(2, 30);
```

Comment:
IIRC, Python now uses "-1" as infinite. Could you confirm?

Reply:
+1

Reply:
Good point. It's actually -1 in python.

```python
        max_restarts (int): Only for *actors*. This specifies the maximum
            number of times that the actor should be restarted when it dies
            unexpectedly. The minimum valid value is 0 (default),
            which indicates that the actor doesn't need to be restarted.
            A value of -1 indicates that an actor should be restarted
            indefinitely.
```
BTW, do we need a constant variable for that in Python?